src/Streamly/Coreutils/Common.hs

```diff
@@ -0,0 +1,49 @@
+module Streamly.Coreutils.Common (
```

Comment:
Remove this module if you are not using it for the changes in this PR.

Reply:
Can I just re-export `Streamly.Coreutils.Uniq`?

Reply:
okay.